models/User.test.ts

```diff
@@ -62,20 +60,23 @@ describe('User', () => {
   });
 
   describe('shouldUpdateOkta', () => {
+    const updateOktaTest = (api: string): void => {
```

Comment:
What prompted this extraction? It doesn't seem like we use it elsewhere and it has a call to `expect` inside which seems a bit strange

Reply:
The original implementation had this error: `Function declared in a loop contains unsafe references to variable(s) 'event', 'user', 'event', 'expect', 'user'. eslint@typescript-eslint/no-loop-func`.

This may be more of a issue specific to React that the backend may not need to care about. What do you think?

Reply:
I think the issue is that we have an it inside of a `for` loop. Does this provide more safety than just one `it` of one api? if so, perhaps moving it to a `forEach` would relieve the issue.

Reply:
I had a revelation as I was thinking about this... this is exactly what jest's `it.each` is for... 😂 fixed it with that